### PR TITLE
Stub UVC camera init

### PR DESCRIPTION
**Pull request by clienthax**
Works enough for S357.

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellCamera.cpp [hunk 1]
```diff
@@ -433,6 +433,15 @@ error_code cellCameraInit()
 		g_camera.attr[CELL_CAMERA_AGCHIGH] = { 64 };
 		break;
 	}
+
+	case fake_camera_type::uvc1_1:
+	{
+		g_camera.attr[CELL_CAMERA_DEVICEID] = { 0x5ca, 0x18d0 };
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellCamera.cpp:439]: what ID is this? comment missing

File: rpcs3/Emu/Cell/Modules/cellCamera.cpp [hunk 2]
```diff
@@ -433,6 +433,15 @@ error_code cellCameraInit()
...
+		g_camera.attr[CELL_CAMERA_FORMATCAP] = { CELL_CAMERA_JPG | CELL_CAMERA_YUV422 };
+		g_camera.attr[CELL_CAMERA_NUMFRAME] = { 1 };
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellCamera.cpp:441]:
```suggestion
		g_camera.attr[CELL_CAMERA_NUMFRAME] = { 1 }; // Amount of supported resolutions
```

File: rpcs3/Emu/Cell/Modules/cellCamera.cpp (hunk 1 above)
Comment (clienthax) [reply rpcs3/Emu/Cell/Modules/cellCamera.cpp:439]: KBCR-S01MU
Seems games check for whatever model they used for a specific cabinet type, so may need to break this out to a config option in future

**Comment by Megamouse** [comment]
I feel like this also needs to implement the getters for CELL_CAMERA_FORMATCAP and CELL_CAMERA_FRAMESIZE.

The getter for CELL_CAMERA_FORMATCAP should return the format that is found using CELL_CAMERA_FORMATINDEX attr index.
For example CELL_CAMERA_FRAMESIZE could be stubbed to return { 640, 480 } for UVC1_1 if CELL_CAMERA_FRAMEINDEX is 0

**Comment by clienthax** [comment]
> I feel like this also needs to implement the getters for CELL_CAMERA_FORMATCAP and CELL_CAMERA_FRAMESIZE.
> 
> The getter for CELL_CAMERA_FORMATCAP should return the format that is found using CELL_CAMERA_FORMATINDEX attr index. For example CELL_CAMERA_FRAMESIZE could be stubbed to return { 640, 480 } for UVC1_1 if CELL_CAMERA_FRAMEINDEX is 0

The game doesn't seem to care currently

File: rpcs3/Emu/Cell/Modules/cellCamera.cpp (hunk 1 above)
Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellCamera.cpp:439]: Comment?

**Pull request merged**